src/pages/execution/workflowTabs/gantt-chart/ConductorTimeline.tsx

```diff
@@ -0,0 +1,274 @@
+
+import {useEffect, useMemo, useState } from 'react';
+import { Button } from '@mui/material';
+import {
+    Bars,
+    Canvas,
+    Cursor,
+    GanttChart,
+    Highlight,
+    Series,
+    XAxis,
+    YAxis,
+  } from './';
+import {HighlightActions} from './HighlightActions';
+import { fontFamily, fontSizes, colors } from './internal/utils';
+import {Datum} from './types'
+
+const font = `${fontSizes.fontSize4} ${fontFamily.fontFamilySans}`;
```

Comment:
No need to clutter file here. Since its a constant style used only once - just inline in the `<GanttChart>` tag